app/src/test/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModelTest.kt

```diff
@@ -390,10 +410,14 @@ class ZimManageViewModelTest {
   }
 
   @Test
-  fun `network states observed`() {
-    networkStates.offer(NOT_CONNECTED)
-    viewModel.networkStates.test()
-      .assertValue(NOT_CONNECTED)
+  fun `network states observed`() = runTest {
```

Comment:
This test will never pass since the `updateNetworkStates` is not called from anywhere.

![Screenshot from 2025-05-23 13-11-36](https://github.com/user-attachments/assets/ea491340-d526-47c2-ba24-7b7a47332596)